### PR TITLE
refactor: New Emoji's Added 😄

### PR DESCRIPTION
**Pull request by adityamangal1**
🦈 Blahaj new emoji's 🦈

### CODE REVIEW

**Review by qwe123coder — APPROVED** [review]
LGTM

**Comment by qwe123coder** [comment]
and change it forme main to dev branch pls

**Comment by riconghost** [comment]
Why is this PR pending ? @neeltron @tyffical

**Comment by tyffical** [comment]
> Why is this PR pending ? @neeltron @tyffical 

'blahajcrying' is not an actual emote in blahajgang server

File: main.py [hunk 1]
```diff
@@ -247,10 +248,10 @@ async def on_message(message):
             'angry': [default_map["angry"]],
             'anger': [default_map["angry"]],
             'mad': [default_map["angry"]],
-            'cry': [custom_map["blahajcry"], custom_map["crii"]],
+            'cry': [custom_map["blahacrying"], custom_map["crii"]],
```
Comment (tyffical) [inline main.py:251]: 'blahajcrying' does not exist

File: main.py [hunk 2]
```diff
@@ -247,10 +248,10 @@ async def on_message(message):
...
             'cri': [custom_map["blahajcry"], custom_map["crii"]],
             'sad': [custom_map["blahajcry"], custom_map["crii"]],
-            'alone': [custom_map["blahajcry"]],
+            'alone': [custom_map["blahacrying"]],
```
Comment (tyffical) [inline main.py:254]: 'blahajcrying' does not exist

**Review by tyffical — CHANGES_REQUESTED** [review]
'blahajcrying' is not an emote on the server. why did u change it from 'blahajcry'?

**Comment by riconghost** [comment]
> 'blahajcrying' is not an emote on the server. why did u change it from 'blahajcry'?

i didnt change anything in the code

**Comment by tyffical** [comment]
Found the error, "blahajcrying" was misspelled on the server. U should change the code to that spelling as well if youd like to keep these reactions

**Comment by adityamangal1** [comment]
> Found the error, "blahajcrying" was misspelled on the server. U should change the code to that spelling as well if youd like to keep these reactions

Will do

**Review by tyffical — APPROVED** [review]
Looks good to me!

**Pull request merged**